src/api/java/baritone/api/utils/ExampleBaritoneControl.java

```diff
@@ -711,4 +719,14 @@ private Goal parseGoal(String[] params) {
         }
         return goal;
     }
+
+    public DimensionType getDimensionByName(String nameIn){ String name = nameIn.toLowerCase();
```

Comment:
what

Reply:
its so you doesnt have to care about capital letters. The new String is because a check wouldnt pass otherwise